app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,23 +1,35 @@
 module Api
   module V1
-    class SessionsController < ApplicationController
-      skip_before_action :authenticate_user!, only: %i[create]
-
+    class SessionsController < ApiController
       def create
-        result = LoginUser.call(session_params)
+        result = LoginUser.call(session_creating_params)
 
         if result.success?
-          render json: { token: result.token }
+          render json: result.to_h.slice(:access_token, :refresh_token)
         else
-          render json: { error: result.error }, status: :unauthorized
+          render json: result.to_h.slice(:error), status: :unauthorized
+        end
+      end
+
+      def update
+        result = RefreshAuthentication.call(session_updating_params)
+
+        if result.success?
+          render json: result.to_h.slice(:access_token, :refresh_token)
+        else
+          render json: result.to_h.slice(:error), status: :unauthorized
         end
       end
 
       private
 
-      def session_params
+      def session_creating_params
         params.require(:session).permit(:email, :password)
       end
+
+      def session_updating_params
+        params.require(:session).permit(:access_token, :refresh_token)
```

Comment:
You do not use access token in the `RefreshAuthentication` interactor if I understand correctly. 
I think you can check that access_token belongs to the same user, as a refresh_token. It will be more secure. What do you think about it?